.evergreen/config.yml

```diff
@@ -1075,6 +1128,14 @@ buildvariants:
            - name: "test-2.4-sharded_cluster"
            - name: "test-2.4-standalone"
 
+- matrix_name: "serverless"
+  matrix_spec:
+    os-fully-featured:
+      - "ubuntu-18.04"
```

Comment:
I wasn't sure what the platform requirement would be for drivers, so I just included a single example.